server/ranking/ranking.go

```diff
@@ -0,0 +1,92 @@
+package ranking
+
+import (
+	"encoding/hex"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"golang.org/x/net/context"
+	
+	"github.com/iotexproject/iotex-election/committee"
+	pb "github.com/iotexproject/iotex-election/pb/ranking"
+)
+
+// server is used to implement pb.RankingServer.
+type server struct {
+	savedCustomers []*pb.RankingServer
+	c              committee.Committee
+}
+
+// GetMeta returns the meta of the chain
+func (s *server) GetMeta(ctx context.Context, empty empty.Empty) (*pb.ChainMeta, error) {
+	height := s.c.LatestHeight()
+	result, err := s.c.ResultByHeight(height)
+	if err != nil {
+		return &pb.ChainMeta{}, err
+	}
+
+	return &pb.ChainMeta{
+		Height:          strconv.FormatUint(height, 10),
+		TotalCandidates: uint64(len(result.Delegates())),
+	}, nil
+}
+
+// GetCandidates returns a list of candidates sorted by weighted votes
+func (s *server) GetCandidates(ctx context.Context, request *pb.GetCandidatesRequest) ([]*pb.CandidateResponse, error) {
```

Comment:
wrong return type, should be *pb.CandidateResponse

Reply:
corrected it. Please see and approve 
https://github.com/iotexproject/iotex-election/pull/5